src/tools/tidy/src/walk.rs

```diff
@@ -31,6 +32,17 @@ pub fn filter_dirs(path: &Path) -> bool {
         "src/bootstrap/target",
         "vendor",
     ];
+    let command =
+        Command::new("git").args(["ls-files", ".", "--exclude-standard", "--others"]).output();
+    if let Ok(output) = command {
+        let stdout: Vec<&str> =
+            String::from_utf8_lossy(&output.stdout).split('\n').into_iter().collect();
+        for line in stdout {
+            if skip.contains(&line) {
+                skip.remove(skip.iter().position(|x| **x = line))
```

Comment:
Shouldn't this just add each line of stdout to `skip`?

Reply:
The issue (unless I'm interpreting it wrong) says that you need to remove these elements from the skip array. But, that could be wrong.

Reply:
> The fix for this is probably to run `git ls-files . --exclude-standard --others` and skip files that it outputs in the default filter_dirs function

The existing `skip` array defines directories that we wish to skip. We want to extend this with any untracked directories - all directories in `skip` are tracked, so right now this wouldn't do anything even without the compile errors